fs/march/march.go

```diff
@@ -416,17 +417,29 @@ func (m *March) processJob(job listDirJob) ([]listDirJob, error) {
 	}
 
 	// If NoTraverse is set, then try to find a matching object
-	// for each item in the srcList
+	// for each item in the srcList with threads
```

Comment:
`srcList with threads` sounds strange

Reply:
sounds good.

Reply:
like this 'for each item in the srcList to head dst object' is ok?